client/src/pages/Bookings/Bookings.tsx

```diff
@@ -0,0 +1,23 @@
+import PageContainer from '../../components/PageContainer/PageContainer';
+import { Grid, Box } from '@mui/material';
+import useStyles from './useStyles';
+import Calendar from '../../components/Calendar/Calendar';
+import 'react-calendar/dist/Calendar.css';
+
+export default function Bookings(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <PageContainer>
+      <Grid container classes={{ root: classes.root }} spacing={{ md: 5 }}>
+        <Grid xs={10} md={4} item>
+          <Calendar />
+        </Grid>
+        <Grid xs={10} md={3} direction="column" item container className={classes.upcomingBookings}>
+          <Box>Your next booking:</Box>
+          <Box>Current bookings:</Box>
```

Comment:
We should implement the UI for booking cards and interacting with them as they are a part of this ticket 👍🏼 I'd recommend using some mock booking data to fill out the respective parts of the UI dynamically still. 
<img width="292" alt="Manage Booking" src="https://user-images.githubusercontent.com/70963670/149496474-a6dc80c8-1055-46b6-ad14-fafa9319845a.png">

Reply:
Will do.